<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Games</class>
 <widget class="QWidget" name="Games">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="bots_layout">
   <property name="topMargin">
    <number>35</number>
   </property>
   <property name="bottomMargin">
    <number>35</number>
   </property>
   <item>
    <widget class="QPushButton" name="games_own">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Checks if current bot instance already owns given appIDs and/or gameNames (can be part of the game's name).&lt;/p&gt;&lt;p&gt;It can also be * to show all games available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Own</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="games_own_all">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Checks if all bots instances already owns given appIDs and/or gameNames (can be part of the game's name).&lt;/p&gt;&lt;p&gt;It can also be * to show all games available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Own All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="games_play">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Switches to manual farming - launches given appIDs on current bot instance.</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
